conda_smithy/templates/azure-pipelines-win.yml.tmpl

```diff
@@ -4,39 +4,8 @@
 
 jobs:
 - job: win
-  pool:
-  {%- for spec in azure.pool_win %}
-    {%- if azure.pool_win[spec] is string %}
-    {{ spec }}: {{ azure.pool_win[spec] }}
-    {%- else %}
-    {{ spec }}:
-    {%- for item in azure.pool_win[spec] %}
-    - {{ item }}
-    {%- endfor %}
-    {%- endif %}
-  {%- endfor %}
-  {%- if azure.workspace is defined %}
-  workspace:
-    clean: {{ azure.workspace.clean }}
-  {%- endif %}
-  timeoutInMinutes: {{ azure.timeout_minutes }}
-  strategy:
-    maxParallel: {{ azure.strategy.maxParallel }}
-    matrix:
-    {%- for data in configs %}
-    {%- if data.platform.startswith('win') %}
-      {{ data.config_name }}:
-        CONFIG: {{ data.config_name }}
-        CONDA_BLD_PATH: D:\\bld\\
-        UPLOAD_PACKAGES: {{ data.upload }}
-    {%- endif %}
-    {%- endfor %}
+  {{ azure_yaml|indent(2) }}
   steps:
-    # TODO: Fast finish on azure pipelines?
-    - script: |
-        ECHO ON
-        {{ fast_finish }}
```

Comment:
You don't need this anymore?

Reply:
afaik this hasn't ever done anything in azure :P